packages/x-adapter-platform/src/types/models/result.model.ts

```diff
@@ -6,10 +6,21 @@ import { PlatformTagging } from './tagging.model';
  * @public
  */
 export interface PlatformResult {
-  id: string;
-  name: string;
-  image: string;
-  price: number;
-  url: string;
+  __id: string;
```

Comment:
Can use add the documentation about these properties? Maybe you can use the one in the `openapi.yml`